lib/Doctrine/ORM/Mapping/UnderscoreNamingStrategy.php

```diff
@@ -58,6 +60,13 @@ class UnderscoreNamingStrategy implements NamingStrategy
      */
     public function __construct($case = CASE_LOWER, bool $numberAware = false)
     {
+        if (! $numberAware) {
+            @trigger_error(
+                'Creating ' . self::class . ' without making it number aware is deprecated and will be removed in Doctrine 3.0.',
```

Comment:
Noticed this in other pull requests, we should be talking about Doctrine **ORM** 3.0 to avoid confusion with other Doctrine libraries.

For consistency, feel free to change this in a separate PR for a subsequent patch release, nothing to hold up 2.7.0 for.

Reply:
Agreed, I almost introduced an inconsistency here. I can quickly address this before releasing.